### ISSUE TITLE
Allow migration to multiple Artifactory repositories

### ISSUE THREAD

**Issue by jfrog-aparicio** [jfrog/docker2artifactory#18]
Currently, when migrating, all images will be placed into the same repository in Artifactory. In some cases, such as when migrating from DTR or Quay, users would like to keep the organization on a per-repository basis.

For example when you have the following in DTR:
myuser/myimage:1.0
myteam/otherimage:1.0

This should result in the creation of the Artifactory Docker repositories myuser and myteam in Artifactory with a single image in each one of them.

In short, this feature will:

1. This feature will be mutually exclusive with the existing repo argument
2. This feature will create Docker repositories in Artifactory if they don't already exist
3. This feature will migrate to multiple DOcker repositories in Artifactory, depending on the name
